src/redux/camera/cameraSlice.ts

```diff
@@ -0,0 +1,32 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface CameraState {
```

Comment:
This name and "subscribeToCamera" seems way too generic IMO, looking at how this is specifically for lat long alt coordinates. 

Please rename it to something more specific? Maybe including geo location or something?

Reply:
Fix later